static/home/js/signprot-multitabtable2.js

```diff
@@ -0,0 +1,1029 @@
+        }
+        catch(err) {
+            column.visible( true, false );
+        }
+    });
+    oTable2.draw();
+}
+
+function reset_tab1() {
+// Just a button to go back to the main page.
+    window.location.href = "/signprot/couplings2";
+}
+
+function reset_tab2() {
+// Just a button to go back to the main page.
+    window.location.href = "/signprot/couplings2";
+}
+
+/**
+ * This is a custom YADCF function that checks ....
+ * ....
+ * @param {object} filterVal Value to filter on (not applicable)
+ * @param {object} columnVal Element in the filtered column
+ * @param {object} rowValues All elements in this row (not used)
+ * @param {object} stateVal Current DOM state of the row (not sufficient in this case)
+ * @returns {boolean} true if row contains selected target otherwise false
+ */
+function supportFilter(filterVal, columnVal, rowValues, stateVal){
+    //console.log(!/^\d+$/.test(columnVal), columnVal, filterVal);
+    //console.log(columnVal === filterVal);
+    return (!/^\d+$/.test(columnVal) || columnVal === filterVal);
+}
+
+/**
+ * Function copied from contactbrowser-tabtables.js
+ * When there's a need to repeat the same yadcf filter_type one can use this function to concatenate
+ * the range_number filter_type.
+ */
+function make_range_number_cols(start_column, repeat_number) {
+    var from_to = {
+        filter_type: "range_number",
+        filter_default_label: ["Min", "Max"],
+        filter_reset_button_text: false
+    };
+    var repeated_from_to = [];
+    for (var i = start_column; i < start_column + repeat_number; i++) {
+        var column_info = Object.assign({}, from_to);
+        column_info["column_number"] = i;
+        repeated_from_to.push(column_info);
+    }
+    return repeated_from_to;
+}
+
+$(function() {
+// Activate tooltips and popovers from Bootstrap   * Bootstrap v3.3.7 (http://getbootstrap.com)
+    $("[data-toggle='tooltip']").tooltip();
+    $("[data-toggle='popover']").popover();
+
+//     $('a[data-toggle="tab"]').on("shown.bs.tab", function (e) {
+// //      console.log( 'show tab' );
+//         $($.fn.dataTable.tables(true)).DataTable()
+//             .columns.adjust().responsive;
+//     });
+
+    console.time("table1load");
+    oTable1 = $("#familiestabletab").DataTable({
+        deferRender: true,
+        scrollY: "50vh",
+        scrollX: true,
+        scrollCollapse: true,
+        scroller: true,
+        paging: false,
+        bSortCellsTop: false, //prevent sort arrows going on bottom row
+        aaSorting: [],
+        order: [[4,"asc"], [22, "desc"]],
+        autoWidth: false,
+        bInfo: true,
+        columnDefs: [
+            {
+                targets: [22],
+                visible: false
+            }
+        ],
+    });
+
+    //Uncheck every row when using back button on browser
+    $(".alt_selected").prop("checked",false);
+    $(".alt").prop("checked",false);
+    $(".select-all").prop("checked",false);
+
+    yadcf.init(oTable1,
```

Comment:
A final suggestion for more concise/succinct coding (not claiming that I do this everywhere 😜). YADCF is heavy on the repetitions for defining the different columns. Example for a bit quicker handling could be to also use for loops here to buildup:
<pre>
for (var i in [6, 7, 8, 9, 10, 14, 17, 20, 22, 24]){
  yadcf_options.push({
      column_number : list[i],
      filter_type: "range_number",
      filter_reset_button_text: false,
      filter_default_label: ["Min", "Max"],
  });
}</pre>

Reply:
I did copy a function from C.Munk
`function make_range_number_cols`
To do this. But then it made it less clear to follow and I reverted to fully explicit.
This suggestion seems better. Will try to try-it out.